.github/workflows/test.yaml

```diff
@@ -76,7 +86,7 @@ jobs:
 
       - name: Run tests
         run: |
-          pytest -k "not acceptance"
+          pytest -k "not acceptance" -vs
```

Comment:
i'm generally pro verbosity (and recommend `-vv` for CI for better observability, especially if/when network things hang), but `-s` is often a bridge too far...

Reply:
```suggestion
          pytest -k "not acceptance" -vv
```